src/main/java/org/mskcc/oncokb/curation/importer/CanonicalTranscriptFlagImporter.java

```diff
@@ -0,0 +1,120 @@
+package org.mskcc.oncokb.curation.importer;
+
+import static org.mskcc.oncokb.curation.util.FileUtils.readDelimitedLinesStream;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.*;
+import java.util.stream.Collectors;
+import org.apache.commons.lang3.StringUtils;
+import org.mskcc.oncokb.curation.domain.Flag;
+import org.mskcc.oncokb.curation.domain.Transcript;
+import org.mskcc.oncokb.curation.domain.enumeration.FlagType;
+import org.mskcc.oncokb.curation.domain.enumeration.ReferenceGenome;
+import org.mskcc.oncokb.curation.service.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+/**
+ * To backfill the OncoKB canonical transcripts
+ */
+
+@Component
+public class CanonicalTranscriptFlagImporter {
+
+    private final Logger log = LoggerFactory.getLogger(CanonicalTranscriptFlagImporter.class);
+
+    final String ONCOKB_GENE_LIST_FILE_PATH = "data/oncokb_core/cancer_gene_list_v4_23.tsv";
```

Comment:
We're on v4.24 for data releases. Did you mean v4.24?

Reply:
Yeah good point. I will update with v4.24